plugin/inc/dataintegrity_tests/tickets_empty_effective_date.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+return [
+    'query' => "
+        select
+            items_id tid
+          , effective_date_field 
+        from glpi_plugin_fields_ticketticketcustomfields
+        where effective_date_field = '' or effective_date_field is null
+        ",
+    'test' => [
+        'alert' => true,
+        'type' => 'compare_query_count',
+        'zero_result' => [
+            'summary_text' => "There are no tickets with empty effective date.",
+        ],
+        'positive_result' => [
+            'summary_text' => "There are {count} tickets with empty effective date.",
+            'iteration_text' => "Ticket <a href='/front/ticket.form.php?id=[tid]' target='_blank'>[tid]</a> has empty effective date.",
```

Comment:
```suggestion
            'iteration_text' => "Ticket <a href='$CFG_PLUGIN_ISERVICE[root_doc]/front/ticket.form.php?id=[tid]' target='_blank'>[tid]</a> has empty effective date.",
```
of course `global $CFG_PLUGIN_ISERVICE[root_doc];` is needed before the return.